boards/bcm2836/Packages/build.txt

```diff
@@ -0,0 +1,19 @@
+@echo off
+
+REM Run this command from the IoTCoreShell prompt only
```

Comment:
this is in iot-adk-addonkit. See https://github.com/ms-iot/iot-adk-addonkit/blob/master/IoTCoreShell.cmd